src/encoded/static/components/app.js

```diff
@@ -1965,7 +1947,7 @@ class ContentErrorBoundary extends React.Component {
     /**
      * Unsets the error state if we navigate to a different view/href .. which normally should be different ContentView.
      */
-    componentWillReceiveProps(nextProps){
+    componentDidUpdate(nextProps){
```

Comment:
Simplest (but not performant) conversion here. Did componentDidUpdate instead of some other update which wouldn't re-render because encountering/having an error ideally would not even happen or be very rare.